docs/topics/forms/formsets.txt

```diff
@@ -448,6 +448,41 @@ happen when the user changes these values::
     {'pub_date': datetime.date(2008, 5, 11), 'ORDER': 1, 'title': 'Article #2'}
     {'pub_date': datetime.date(2008, 5, 10), 'ORDER': 2, 'title': 'Article #1'}
 
+``ordering_widget``
+-------------------
```

Comment:
We need to use [subsubsections](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html?highlight=admonition#sections) here. 

Please change the `---` line for `^^^` characters. (Same below)

(This is used elsewhere: `egrep -r -e '^\^+' docs`)